### PR TITLE
PLAT-712: update deduplication tests

### PR DESCRIPTION
**Pull request by MariaVasilenko**
update cases and tests
https://insolar.testrail.io/index.php?/cases/view/5096
https://insolar.testrail.io/index.php?/cases/view/5095

### CODE REVIEW

File: ledger-core/virtual/integration/deduplication_test.go [hunk 1]
```diff
@@ -35,192 +35,254 @@ func TestDeduplication_SecondCallOfMethodDuringExecution(t *testing.T) {
...
+			if test.countVFindCallRequest == 0 {
+				outgoing = server.BuildRandomOutgoingWithPulse()
+			}
 
-	outgoing := helper.BuildObjectOutgoing()
+			releaseBlockedExecution := make(chan struct{}, 0)
+			numberOfExecutions := 0
+			// Mock
+			{
+				isolation := contract.MethodIsolation{Interference: contract.CallTolerable, State: contract.CallDirty}
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/deduplication_test.go:87]: tolerableFlags()

File: ledger-core/virtual/integration/deduplication_test.go [hunk 2]
```diff
@@ -35,192 +35,254 @@ func TestDeduplication_SecondCallOfMethodDuringExecution(t *testing.T) {
...
-	{
-		isolation := contract.MethodIsolation{Interference: contract.CallTolerable, State: contract.CallDirty}
-		runnerMock.AddExecutionClassify("SomeMethod", isolation, nil)
+			oneExecutionEnded := server.Journal.WaitStopOf(&execute.SMExecute{}, 1)
+			executeDone := server.Journal.WaitStopOf(&execute.SMExecute{}, 2)
 
-		newObjDescriptor := descriptor.NewObject(
-			reference.Global{}, reference.Local{}, class, []byte(""), false,
-		)
+			runnerMock := logicless.NewServiceMock(ctx, mc, func(execution execution.Context) string {
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/deduplication_test.go:62]: You can use default ServiceMock

File: ledger-core/virtual/integration/deduplication_test.go [hunk 3]
```diff
@@ -35,192 +35,254 @@ func TestDeduplication_SecondCallOfMethodDuringExecution(t *testing.T) {
...
+			if test.countVFindCallRequest == 0 {
+				outgoing = server.BuildRandomOutgoingWithPulse()
+			}
 
-		return false
-	}).ExpectedCount(2)
+			numberOfExecutions := 0
+			// Mock
+			{
+				isolation := contract.MethodIsolation{Interference: contract.CallTolerable, State: contract.CallDirty}
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/deduplication_test.go:208]: tolerableFlags()

File: ledger-core/virtual/integration/deduplication_test.go [hunk 4]
```diff
@@ -35,192 +35,254 @@ func TestDeduplication_SecondCallOfMethodDuringExecution(t *testing.T) {
...
-	{
-		isolation := contract.MethodIsolation{Interference: contract.CallTolerable, State: contract.CallDirty}
-		runnerMock.AddExecutionClassify("SomeMethod", isolation, nil)
+			server, ctx := utils.NewUninitializedServer(nil, t)
+			defer server.Stop()
 
-		newObjDescriptor := descriptor.NewObject(
-			reference.Global{}, reference.Local{}, class, []byte(""), false,
-		)
+			runnerMock := logicless.NewServiceMock(ctx, mc, func(execution execution.Context) string {
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/deduplication_test.go:183]: You can use a default ServiceMock.

File: ledger-core/virtual/integration/deduplication_test.go (hunk 3 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/deduplication_test.go]: done

File: ledger-core/virtual/integration/deduplication_test.go (hunk 1 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/deduplication_test.go]: done

File: ledger-core/virtual/integration/deduplication_test.go (hunk 2 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/deduplication_test.go]: done

File: ledger-core/virtual/integration/deduplication_test.go (hunk 1 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/deduplication_test.go]: done

File: ledger-core/virtual/integration/deduplication_test.go (hunk 2 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/deduplication_test.go]: done

File: ledger-core/virtual/integration/deduplication_test.go (hunk 3 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/deduplication_test.go]: done

File: ledger-core/virtual/integration/deduplication_test.go (hunk 4 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/deduplication_test.go]: done

**Pull request merged**